source/common/http/conn_manager_impl.h

```diff
@@ -963,11 +1083,11 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     }
 
     ConnectionManagerImpl& connection_manager_;
+    const uint64_t stream_id_;
```

Comment:
looks like this is in 2 places.  candidate for future clean up?

Reply:
Conceptually it feels like this belongs on the ActiveStream, but at this point the main use case of it in AS is for `ENVOY_STREAM_LOG`. I guess one option would be to have the AS allocate the stream id and pass it to the FM, then refer to the FM when doing stream logging from AS?

Reply:
I was just thinking one could snag it from the other via accessor / callback.
Take a look and if you think it makes sense you can do in a follow-up?

Reply:
Yeah I'll see if I can find something that makes sense